db/schema.rb

```diff
@@ -375,6 +375,7 @@
     t.integer "vbms_upload_failure_count", default: 0
     t.integer "bgs_upload_failure_count", default: 0
     t.string "claim_id"
+    t.integer "tracked_items", default: [], array: true
   end
```

Comment:
@stiehlrod This line right here makes sense, I'm not entirely sure what to make of all the other changes.  Do you know, was that something that came up a version thing, or some other issue when you were setting this up?  Just looking for some context.

Reply:
@rockwellwindsor-va I am pretty sure I am not supposed to add the schema, but I was getting errors so I tried it. I just couldn't remember what Platform prefers, but now I am convinced that we shouldn't push the schema. If you run rails db:migrate you will see all of those changes too.

Reply:
@rockwellwindsor-va https://dsva.slack.com/archives/CBU0KDSB1/p1721751714314029